core/server/worker/src/test/java/alluxio/worker/block/SpaceReserverTest.java

```diff
@@ -41,8 +38,7 @@
 /**
  * Unit tests for {@link SpaceReserver}.
  */
-@RunWith(PowerMockRunner.class)
-@PrepareForTest({BlockWorker.class, BlockStoreMeta.class})
+@RunWith(MockitoJUnitRunner.class)
```

Comment:
I think it can validate the usage of Mockito and keep the same format with the old file. So I add this.